privacyidea/lib/policy.py

```diff
@@ -2016,7 +2017,11 @@ def get_static_policy_definitions(scope=None):
                           "attribute, set this to '*'. Use '*' with CAUTION! "
                           "For more details, check the documentation."),
                 'mainmenu': [],
-                'group': GROUP.USER}
+                'group': GROUP.USER},
+            ACTION.TOKENINFO_API_EXCLUDE: {
```

Comment:
Please add this action to SCOPE.USER and SCOPE.ADMIN, and rename it, analog to `HIDE_AUDIT_COLUMNS` to `HIDE_TOKENINFO`, see:

https://github.com/privacyidea/privacyidea/blob/a6c8ce47fdf8746d25f2e4f3c2a9b7478043f820/privacyidea/lib/policy.py#L1749